_data/authors.yml

```
@@ -560,6 +560,12 @@ mitchellallison:
   github: mitchellallison
   about: "Mitchell Allison works on Distributed Systems in Swift at Apple."
 
+ahmedelrefaey:
+  name: Ahmed Elrefaey
+  email: a7med.mahmoud2004@gmail.com
+  github: a7medev
+  about: "Ahmed is a Google Summer of Code 2025 contributor, where he worked on improving the display of Swift documentation during code completion in SourceKit-LSP and VS Code."
```

Sorry I didn't suggest this earlier. 😅

```suggestion
  about: "Ahmed is a software developer at Luciq. During Google Summer of Code 2025, he worked on improving the display of Swift documentation during code completion in SourceKit-LSP and VS Code."
```